modules/artifact-registry/README.md

```diff
@@ -100,14 +101,50 @@ module "registry-maven" {
 
 # tftest modules=2 resources=2
 ```
+
+## Docker Cleanup Policies
```

Comment:
Change to just "Cleanup Policies" (cleanup policies apply to all formats except Apt and Yum)

Reply:
fixed